clients/bazel-module-registry/src/main/kotlin/Model.kt

```diff
@@ -44,6 +44,7 @@ data class ModuleMetadata(
     @Serializable
     data class Maintainer(
         val email: String? = null,
+        val github: String? = null,
```

Comment:
What's the benefit of adding this here if it's unused and we have `ignoreUnknownKeys = true`?

Reply:
Consistency to support the most important fields of the model that could be used, in the future?

Please none that the `Maintainer` class is already unused.

Reply:
> Consistency to support the most important fields of the model that could be used, in the future?

Then please add that as a motivation to the commit message. Otherwise, as a maintainer I would be looking for the use of that field, thinking that I overlooked something if I don't find any.